src/common/services/remote-config/types.ts

```diff
@@ -104,7 +109,12 @@ export enum BooleanKeys {
   /**
    * Boolean to use amplitude as the metrics tracking.
    */
-  USE_AMPLITUDE = 'USE_AMPLITUDE'
+  USE_AMPLITUDE = 'USE_AMPLITUDE',
+
+  /**
+   * Boolean to use solana wrapped audio.
```

Comment:
Nit: use in what sense? Would be nice to explain in comments that this is for which token we prefer in transfer etc